fe/fe-core/src/main/java/com/starrocks/connector/paimon/PaimonConnector.java

```diff
@@ -76,8 +75,8 @@ public PaimonConnector(ConnectorContext context) {
         }
         paimonOptions.setString(WAREHOUSE.key(), warehousePath);
 
-        if (this.cloudConfiguration.getCloudType() == CloudType.AWS && this.cloudConfiguration instanceof AWSCloudConfiguration) {
-            AWSCloudConfiguration awsCloudConfiguration = (AWSCloudConfiguration) this.cloudConfiguration;
+        if (cloudConfiguration.getCloudType() == CloudType.AWS && cloudConfiguration instanceof AWSCloudConfiguration) {
```

Comment:
`if (cloudConfiguration.getCloudType() == CloudType.AWS)` is enough, that means cloudConfiguration must be AWSCloudConfiguration

Reply:
OK. fixed.